### DISCUSSION TITLE
new slot: lat_uncertainty, lon_uncertainty , lat_lon sometimes has uncertainty measurements with is

### DISCUSSION THREAD

**Discussion by mslarae13** [microbiomedata/nmdc-schema#2310] [Schema Best Practices]
Some projects capture a level of uncertainty for lat_lon.

Should NMDC have a slot to capture this?

Pros:
- More metadata
- more consistent & accurate
- Aligns with projects that capture this

Cons:
- more slots
- not commonly used/ provided
- how would we use this? user research not completed on how useful it would be

**Comment by ssarrafan** [comment]
@mslarae13 can this be closed with the release of Berkeley schema?

**Comment by mslarae13** [comment]
Berkeley schema did not address this. 
But this should be a discussion until a decision is made.
